.github/workflows/ci.yml

```diff
@@ -4,11 +4,11 @@ on:
   push:
     branches:
       - main
-      - 'release/**'
+      - 'release**'
   pull_request:
     branches:
       - main
-      - 'release/**'
+      - 'release**'
```

Comment:
I think you should remove the two stars as there will only be the one and only 'release' branch from now on.

Reply:
Don't we want to build our `release-1.2.3_asd` PRs?

Reply:
Ah, but those PRs target ether `main` or `release`.

U are right.